src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -132,6 +139,127 @@ public function getCrossingChar()
         return $this->crossingChar;
     }
 
+    /**
+     * Sets crossing characters.
+     *
+     * Example:
+     * <code>
+     * 1---------------2-----------------------2------------------3
+     * | ISBN          | Title                 | Author           |
+     * 8---------------0-----------------------0------------------4
+     * | 99921-58-10-7 | Divine Comedy         | Dante Alighieri  |
+     * | 9971-5-0210-0 | A Tale of Two Cities  | Charles Dickens  |
+     * | 960-425-059-0 | The Lord of the Rings | J. R. R. Tolkien |
+     * 7---------------6-----------------------6------------------5
+     * </code>
+     *
+     * @param string      $cross        Crossing char (see #0 of example)
+     * @param string|null $topLeft      Top left char (see #1 of example), equal to $cross if undefined
+     * @param string|null $topMiddle    Top middle char (see #2 of example), equal to $cross if undefined
+     * @param string|null $topRight     Top right char (see #3 of example), equal to $cross if undefined
+     * @param string|null $middleRight  Middle right char (see #4 of example), equal to $cross if undefined
+     * @param string|null $bottomRight  Bottom right char (see #5 of example), equal to $cross if undefined
+     * @param string|null $bottomMiddle Bottom middle char (see #6 of example), equal to $cross if undefined
+     * @param string|null $bottomLeft   Bottom left char (see #7 of example), equal to $cross if undefined
+     * @param string|null $middleLeft   Middle left char (see #8 of example), equal to $cross if undefined
+     *
+     * @return $this
+     */
+    public function setCrossingChars(string $cross, string $topLeft = null, string $topMiddle = null, string $topRight = null, string $middleRight = null, string $bottomRight = null, string $bottomMiddle = null, string $bottomLeft = null, string $middleLeft = null)
+    {
+        $this->crossingChar = $cross;
+        $this->crossingTopLeftChar = $topLeft ?? $cross;
+        $this->crossingTopMiddleChar = $topMiddle ?? $cross;
+        $this->crossingTopRightChar = $topRight ?? $cross;
+        $this->crossingMiddleRightChar = $middleRight ?? $cross;
+        $this->crossingBottomRightChar = $bottomRight ?? $cross;
+        $this->crossingBottomMiddleChar = $bottomMiddle ?? $cross;
+        $this->crossingBottomLeftChar = $bottomLeft ?? $cross;
+        $this->crossingMiddleLeftChar = $middleLeft ?? $cross;
+
+        return $this;
+    }
+
+    /**
+     * Gets crossing top right char.
+     *
+     * @return string $crossingTopRightChar
+     */
+    public function getCrossingTopRightChar()
```

Comment:
Return type-hints in these methods?

```php
public function getCrossingTopRightChar(): string { ... }
```

Reply:
Yes it would be better, but the others getters of this class don't have return type-hints.

Reply:
not a big deal IMO, let's use new PHP features in new code

Reply:
Done :)